### ISSUE TITLE
cobble spillage with automated geothermal generator

### ISSUE THREAD

**Issue by wallasaurus** [DarkPacks/SkyFactory-4#1365]
When using vanilla dropper to put cobble into glass cauldron, triggered by redstone clock, over time due to unclear reasons the cobble can 'miss' the cauldrons and begin to accumulate, eventually causing the server to have issues and flatline the CPU.

I've since changed over to an autoclider to put the cobble direct to lava, otherwise i'd give you a more detailed printscreen-accompanied diagram, but I'll note here all that I can recall.

cauldron with comparator and length 4 redstone trail, signal inverter and redstone clock set to respond to redstone.  Redstone closk was 5 duration, 40 or so delay between pulses.  Was all working as expected, shutting off when cauldrons were full.  diamond cobble generator feeding via item extractor + cable to vanilla droppers which emit downwards into the cauldrons.  Lava fed by fluid extractor and cable to geothermal generator.

Not sure why it does this, but it seems to work when observed - i even got a second opinion in case i was crazy or missed something, but when logged out it was causing issues.  The area was not even chunk loaded (not that I know of) so I am not sure how it even did this when I was not logged in.  Maybe some other block acts as a chunk loader?

Sorry it is not more detailed report.  :(  Good luck!

**Comment by Searchcat37** [comment]
This happens to me but to fix this I have clear lag to get rid items.

**Comment by KollinsPlays** [comment]
The way the cauldron works if you dispense items instead of clicking them in, it occasionally checks for entities that have landed inside and then accepts them as input. This is the same thing a TiCo smeltery does when you drop stuff in the multiblock instead of putting stuff into the controller. Actually, the same thing happens in vanilla if you dispense an item onto a pressure plate and it misses. I don't think it's a bug of SF4 per say, because the problem is when the cobble is just entities and haven't yet been accepted by the cauldron or land outside it's accepted input area, it's just an empty cauldron with no comparator signal and also has no control over what entities are on the ground. You would have to adjust your dispenser to not drop cobble too fast or miss the cauldron. The same thing would happen if you dispense into a smeltery and miss or drop more entites than it has room for. This video http://youtu.be/8h37-skZx_g uses Integrated Dynamics as a dispenser, but has the same problem if you dispense cobble too fast.

TL;DR this isn't a bug in the mod pack, it's a problem with using the core Minecraft mechanic of dropping entities into the world and hoping they land and register on/in the right place. The fix is to adjust the way you dispense cobble or use one of the click methods like the auto clicker or automated user.

**Comment by wallasaurus** [comment]
Oki doki – I changed stuff around and use one of the clicker blocks and that works fine, the only reason it was strange was that it was not chunk loaded as far as I was aware, so I didn’t know why it happened when I was offline.   Anyhow, no problem  😊  Thanks for looking into it.
